### ISSUE TITLE
[server] wordserver crash

### ISSUE THREAD

**Issue by mpfans** [azerothcore/azerothcore-wotlk#11296]
### Current Behaviour

crash log
https://gist.github.com/mpfans/0ef2ba09e560a29d73f2555d0ccfc27a

### Expected Blizzlike Behaviour

none

### Steps to reproduce the problem

none

### AC rev. hash/commit

285504e79d25c84b88413fdfad17c2b83fefda86

### Operating system

ubuntu 20.04

**Comment by pklloveyou** [comment]
My server crashed, too, just like yours

**Comment by Kitzunu** [comment]
crash here https://github.com/azerothcore/azerothcore-wotlk/blob/6efc08f42e8ae731776517b187a79ba2ad546ec7/src/server/game/AuctionHouse/AuctionHouseMgr.cpp#L892

cc @UltraNix

**Comment by UltraNix** [comment]
@Kitzunu I don't see anything specific here.
@mpfans That crash occured only once?

**Comment by UltraNix** [comment]
@mpfans Need more about this.

**Comment by mpfans** [comment]
> @Kitzunu I don't see anything specific here. @mpfans That crash occured only once?

yes.. only once ... but  crash now . I don't know if it's the same reason

https://github.com/azerothcore/azerothcore-wotlk/issues/11347

**Comment by UltraNix** [comment]
Just show me crashlog and I will tell you.

**Comment by UltraNix** [comment]
Duplicate of #11281

**Comment by UltraNix** [comment]
So let's wait until it crashes again.
